packages/design-system/src/components/IdleTimeout/useInterval.js

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useRef } from 'react';
+
+// https://overreacted.io/making-setinterval-declarative-with-react-hooks/
```

Comment:
For when this disappears some day, is there any way usage could be synopsized in 1 to 2 lines?  😰